lib/jmap/mdn/method/send/send_message_method.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:jmap_dart_client/http/converter/account_id_converter.dart';
+import 'package:jmap_dart_client/http/converter/identities/identity_id_converter.dart';
+import 'package:jmap_dart_client/http/converter/mdn/mdn_converter.dart';
+import 'package:jmap_dart_client/http/converter/set/set_method_properties_converter.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/capability/capability_identifier.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/method/method.dart';
+import 'package:jmap_dart_client/jmap/core/patch_object.dart';
+import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:jmap_dart_client/jmap/mdn/mdn.dart';
+
+class SendMessageMethod extends MethodRequiringAccountId {
+  final IdentityId identityId;
+
+  final Map<Id, Mdn> send;
+
+  final Map<Id, PatchObject>? onSuccessUpdateEmail;
+
+  SendMessageMethod(
+    AccountId accountId, {
+    required this.identityId,
+    required this.send,
+    this.onSuccessUpdateEmail,
+  }) : super(accountId);
+
+  @override
+  MethodName get methodName => MethodName('MDN/send');
+
+  @override
+  Set<CapabilityIdentifier> get requiredCapabilities => {
+        CapabilityIdentifier.jmapMdn,
+        CapabilityIdentifier.jmapMail,
```

Comment:
As the documentation describes `MDN`, we need to add core capability `urn:ietf:params:jmap:core`

Reply:
i have read the document again, but i don't see where it describe `MDN` need to have `urn:ietf:params:jmap:core`. Can you show me what document is, please? 
![image](https://user-images.githubusercontent.com/43041967/194214415-16b72db8-cea1-4294-8ab9-050b0cd7fff7.png)